packages/test/test-end-to-end-tests/src/test/messageSize.spec.ts

```diff
@@ -321,31 +321,34 @@ describeCompat("Message size", "NoCompat", (getTestObjectProvider, apis) => {
 			},
 		).timeout(chunkingBatchesTimeoutMs);
 
-		describe(`Large payloads (exceeding the 1MB limit) - ${
+		describe(`Large payloads - ${
```

Comment:
So do we not need tests that test for >1MB payload?

Reply:
There are still a couple tests that verify simple behavior of sending large payloads (rejected vs accepted)